src/EventService.Mappers/Db/DbEventMapper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using LT.DigitalOffice.EventService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.EventService.Models.Db;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.Event;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.EventService.Mappers.Db;
+
+public class DbEventMapper : IDbEventMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public DbEventMapper(IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public DbEvent Map(CreateEventRequest request)
+  {
+    Guid eventId = Guid.NewGuid();
+
+    return request is null
+      ? null
+      : new DbEvent
+      {
+        Id = eventId,
```

Comment:
you use Guid from line 21 just once here => don't need to create variable
```
  public DbEvent Map(CreateEventRequest request)
  {
    return request is null
      ? null
      : new DbEvent
      {
        Id = Guid.NewGuid(),
```

Reply:
Done

Reply:
Please, don't close the conversations still make changes